tensorflow/python/keras/distribute/distribute_strategy_test.py

```diff
@@ -689,6 +697,28 @@ def test_numpy_with_sample_weights(self, distribution):
       model.fit(inputs, targets, sample_weight=sample_weights, epochs=1,
                 steps_per_epoch=2, verbose=1)
 
+  @combinations.generate(
+      combinations.combine(distribution=strategies_minus_tpu, mode=['graph']))
+  def test_sample_weights(self, distribution):
+    with self.cached_session():
+      with distribution.scope():
```

Comment:
nit: this can be inlined with the previous with statement.

Reply:
Sorry, I couldn't quite understand what do you mean above, could you please give me some more specific explanation. Thanks.

Reply:
nit:I think you can say:
"with self.cached_session(), distribution.scope():"

Reply:
I see, Thanks, just want to be consistent with the test above/below. Now I have changed it.